exploitthrower.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import swpag_client
+import pwn
+import subprocess
+
+URL = "52.52.83.248" #CHANGE THIS
+TEAM_URL = "http://" + URL + "/"
+TEAM_FLAG_TOKEN = "ndiVzdmIxXjyyXbRbVUj"#CHANGE THIS
```

Comment:
Be careful not to leak things like this in the actual event because this repo is public.